src/theme/shared_variables.scss

```diff
@@ -189,6 +189,7 @@ $spacing-xl: $spacing-l * 2;
   --color-primary-contrast-rgb: 255, 255, 255;
   --color-primary-shade: #3e9792;
   --color-primary-tint: #59b4af;
+  --color-primary-lime: #DDF3E9;
```

Comment:
Geen tinten bijmaken ajb. Je kan hier ofwel met opacity werken als het nodig is. Er bestaat ook al een box-shadow variabele, mss kan je die gewoon hergebruiken.